src/url-loader/url-resolver.ts

```diff
@@ -65,3 +70,25 @@ export abstract class UrlResolver {
     return url as ResolvedUrl;
   }
 }
+
+async function inferComponentDirname(dirname: string): Promise<string> {
+  try {
+    const contents = await fs.readFile(
+        pathlib.join(dirname, '.bowerrc'), {encoding: 'utf8'});
```

Comment:
If I recall, Bower has some particular method of finding a .bowerrc file, and there's a library you can use to find it. It might just be a findup-style lookup.